На прорверку.

Poteryashka163 opened the pull request.

Commit:
На прорверку.

artem-petrosyan commented on pom.xml:
```diff
@@ -22,13 +22,54 @@
+			<version>42.3.1</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc11</artifactId>
+			<version>4.3</version>
+		</dependency>
```
Зависимости для oracle не нужны. Можно удалить.

Commit:
На прорверку задания 3.4 "Введение в SQL"

artem-petrosyan commented on src/main/java/ru/hogwarts/school/model/Faculty.java:
```diff
@@ -25,37 +50,19 @@ public boolean equals(Object o) {
-        return id;
-    }
+    @OneToMany(mappedBy = "faculty", fetch = FetchType.LAZY)
+    private List<Student> students;
```
Нужно повесить аннотацию `@JsonIgnore`
Иначе упадём с исключением.
Также подсвечу, что поля класса лучше располагать до конструкторов и методов.

artem-petrosyan commented on scripts.sql:
```diff
@@ -0,0 +1,6 @@
+select * from student
+select name from student
+where  age>10 and age<20
+where name like '%о%'
+WHERE age>id
+ORDER BY age
```
Напиши, пожалуйста, запросы целиком. Вместе с select'ом

Commit:
На прорверку задания 3.4 "Введение в SQL" добавлен swagger

Commit:
На прорверку задания 3.4 "Введение в SQL" добавлен swagger
Исправлено:
-удалена зависимость oracle
-запросы sql запитаны полностью
-добавлена аннотация @JsonIgnore
-поля класса расположены на уровне экземплярных полей

Poteryashka163 committed:
добавлено исправление в классе Student

Commit:
На прорверку(от 06.04) задания 3.4 "Введение в SQL" добавлен swagger
Исправлено:
-удалена зависимость oracle
-запросы sql запитаны полностью
-добавлена аннотация @JsonIgnore
-поля класса расположены на уровне экземплярных полей

artem-petrosyan commented on src/main/java/ru/hogwarts/school/model/Faculty.java:
```diff
@@ -1,21 +1,51 @@
     private String color;
 
+
+    @OneToMany(mappedBy = "faculty", fetch = FetchType.LAZY)
+    private List<Student> students;
```
Должно быть вот так:
```
    @OneToMany(mappedBy = "faculty", fetch = FetchType.LAZY)
    @JsonIgnore
    private List<Student> students;
```

artem-petrosyan commented on src/main/java/ru/hogwarts/school/model/Student.java:
```diff
@@ -1,17 +1,21 @@
-        this.name = name;
-        this.age = age;
-    }
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "faculty_id")
+    @JsonIgnore
+    private Faculty faculty;
```
Должно быть вот так:
```
    @ManyToOne
    @JoinColumn(name = "faculty_id")
    private Faculty faculty;
```

Commit:
На прорверку задания 3.4(решение задания 3.5)
